packages/swap/src/utils/math.ts

```diff
@@ -0,0 +1,9 @@
+import BigNumber from 'bignumber.js';
+
+export const percentDiff = (expected: BigNumber.Value, actual: BigNumber.Value) => {
+  const a = new BigNumber(expected);
+  const b = new BigNumber(actual);
+
+  // |a - b| / ((a + b) / 2) * 100
```

Comment:
why do we use the average here?

Reply:
It's the formula to calculate the percent difference between two values. The value hasn't change from anything to something else, they're just different values.

Reply:
im not sure about this, right now its the percent difference relative to the average of the two values